frontend/src/app/playback/playback.component.html

```diff
@@ -1,24 +1,17 @@
-<div class="container">
-  <div *ngIf="isLoading; else content" class="loading-message">
-    <p>{{ message }}</p>
+@if (isLoading) {
```

Comment:
As of Angular 17, you can use this new `@if` and `@for` control flow syntax. Some people prefer the pre-Angular 17 way you have it, but just throwing it out there if you weren't aware.

It's a bit less code without the extra `<div>`'s and `<ng-template>`'s. Using the new syntax apparently slightly reduces the app's bundle size a bit too: https://blog.angular-university.io/angular-if/